### PR TITLE
Unit 9: Model Optimization (Model optimization tools and framework)

### PR DESCRIPTION
**Pull request by haizadtarik**
Add overview and setup guides for the following model optimization tools:
- Tensorflow Model optimization Toolkit (TMO)
- Pytorch Quantization
- ONNX runtime
- tensorrt
- openvino
- edgeTPU

### CODE REVIEW

**Comment by haizadtarik** [comment]
created PR for the overview page first. Example notebooks is still WIP

File: chapters/en/Unit 9 - Model Optimization/tools_and_frameworks.mdx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+The Tensorflow Model Optimization Toolkit is available as a pip package, `tensorflow-model-optimization`. To install the package, run the following command:
+```
+pip install -U tensorflow-model-optimization
+```
+
+## Pytorch Quantization
+
+### Overview
+
+For optimizing model, pyTorch supports INT8 quantization compared to typical FP32 models which leads to 4x reduction in the model size and a 4x reduction in memory bandwidth requirements.
```
Comment (merveenoyan) [inline chapters/en/Unit 9 - Model Optimization/tools_and_frameworks.mdx:23]:
```suggestion
For optimizing model, PyTorch supports int8 quantization compared to typical FP32 models, which leads to a 4x reduction in the model size and a 4x reduction in memory bandwidth requirements. 
```

File: chapters/en/Unit 9 - Model Optimization/tools_and_frameworks.mdx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+pip install -U tensorflow-model-optimization
+```
+
+## Pytorch Quantization
+
+### Overview
+
+For optimizing model, pyTorch supports INT8 quantization compared to typical FP32 models which leads to 4x reduction in the model size and a 4x reduction in memory bandwidth requirements. 
+PyTorch supports multiple approaches to quantizing a deep learning model which are as follows:
+1. Model is trained in FP32 and then the model is converted to INT8.
```
Comment (merveenoyan) [inline chapters/en/Unit 9 - Model Optimization/tools_and_frameworks.mdx:25]:
```suggestion
1. Model is trained in FP32 and then the model is quantized to INT8. 
```

**Review by merveenoyan — APPROVED** [review]
Looks good, thanks!

**Review by ATaylorAerospace — APPROVED** [review]
This Unit looks great.

**Pull request merged**